packages/KMenu/KMenuItem.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <div
+    :data-testid="item ? `${item.title.replace(' ', '-')}-menu-item` : 'menu-item'"
+    :test-mode="testMode"
+    :class="[isOpen && expandable ? 'title-dark' : '', {'expand-item' : expandable}]"
+    aria-labelledby="menu-item"
```

Comment:
I can't tell what this is associated with. It should be the id of an element.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Reply:
Fixed.